homeassistant/components/awair/quality_scale.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: todo
+  config-flow: done
+  config-flow-test-coverage: todo
```

Comment:
`test_show_form` can be merged with the happy flows to create a cloud and local config entry, please also move the happy flows to the top